modules/realm_core/include/realm_core/analysis.h

```diff
@@ -36,15 +36,15 @@ namespace analysis
  * @param flag Color layout
  * @return Colormap of input mat
  */
-cv::Mat convertToColorMapFromCVFC1(const cv::Mat &img, const cv::Mat &mask, cv::ColormapTypes flag);
+cv::Mat convertToColorMapFromCVC1(const cv::Mat &img, const cv::Mat &mask, cv::ColormapTypes flag);
 
 /*!
  * @brief Converts a three channel floating point mat to a RGB color map
  * @param img CV_32FC3 or CV_64FC3 floating point mat
  * @param mask Mask of valid pixels
  * @return Colormap of input mat
  */
-cv::Mat convertToColorMapFromCVFC3(const cv::Mat &img, const cv::Mat &mask);
+cv::Mat convertToColorMapFromCVC3(const cv::Mat &img, const cv::Mat &mask);
```

Comment:
What are these changes?

Reply:
Those were related to being able to write out the normals and depth images.  They were being fed to the CVF3 function that did not accept unsigned int matrices.  

https://github.com/laxnpander/OpenREALM/blob/363e4e955382f62d5822ac2c2ed869bad4a2bcb1/modules/realm_io/src/cv_export.cpp#L190-L199

Alternately, I could spin off a separate function, but the function contents itself would be the same, just with a different assert check at the beginning.

Reply:
For further reference, it was all contained in this commit:
https://github.com/laxnpander/OpenREALM/pull/45/commits/164282a8c5cb90ea0f69932a1973783ffb59c061

Basically, the Number of Observations wasn't able to save to the debug output since it was failing the assert statement.  I decided to expand the capabilities of the color map saving function to allow floating or unsigned values as the fix.  If you would prefer separate functions, I'm not against that approach either.

I had forgotten about this change, and it really should have been in a separate pull request.

Reply:
I'm hoping to rebase after this pull request and #37.  Those are the bigger ones to get it up and running.  After that it should be smaller adjustments / improvements.